### DISCUSSION TITLE
最新版 3.9.10.19，注入dll之后，微信会弹出黑框，收不到HOOK消息

### DISCUSSION THREAD

**Discussion by WIGIW1314** [ttttupup/wxhelper#423] [Q&A]
最新版 3.9.10.19，注入编译好的dll之后，微信会弹出黑框，一旦关闭微信也跟着关了，是什么问题？而且收不到HOOK消息，其他功能正常
![image](https://github.com/ttttupup/wxhelper/assets/168173068/64c1f19e-3f68-49bf-95eb-0f071fc9b518)

**Comment by Chener-03** [comment]
![image](https://github.com/ttttupup/wxhelper/assets/68767963/88f85724-e0a8-44ce-8b25-baa5751e3c7b)

**Comment by ttttupup** [comment]
debug 模式启动会打开控制台窗口
